### PR TITLE
ENTESB-5154  fuse-bxms-integ checkstyle of the code

### PR DESCRIPTION
**Pull request by dvirgiln**
https://issues.jboss.org/browse/ENTESB-5154

### CODE REVIEW

File: camel/kie-camel/src/main/java/org/kie/camel/component/ExecutionNodePipelineContextImpl.java [hunk 1]
```diff
@@ -18,26 +18,59 @@
 
 import org.kie.api.runtime.CommandExecutor;
 
+
+/**
+ * The Class ExecutionNodePipelineContextImpl.
+ */
```
Comment (psiroky) [inline camel/kie-camel/src/main/java/org/kie/camel/component/ExecutionNodePipelineContextImpl.java]: So the checkstyle now enforces javadoc on every class, method and field?

Comment (dvirgiln) [inline camel/kie-camel/src/main/java/org/kie/camel/component/ExecutionNodePipelineContextImpl.java]: yes

File: camel/kie-camel/src/main/java/org/kie/camel/component/ExecutionNodePipelineContextImpl.java [hunk 2]
```diff
@@ -18,26 +18,59 @@
...
     public void setCommandExecutor(CommandExecutor exec) {
         this.exec = exec;
     }
 
+    /**
+     * Gets the local class loadel.
+     *
+     * @return the local class loadel
+     */
     public ClassLoader getLocalClassLoadel() {
```
Comment (psiroky) [inline camel/kie-camel/src/main/java/org/kie/camel/component/ExecutionNodePipelineContextImpl.java]: This looks like a typo (which was already there though).

**Comment by psiroky** [comment]
-1. I think we should not commit auto-generated javadoc which adds no additional value. Better to have no javadoc in my opinion and over time add a quality javadoc which describes behavior that is not apparent from the code (that is imho the real value of javadoc).

**Comment by psiroky** [comment]
I am not able to go through the whole diff as it is huge. But assuming the generated javadocs were removed, then +1 to merge.

**Pull request merged**